src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -397,8 +400,12 @@ public static CompletedBuildResponse Create(BinaryReader reader)
             var utf8Output = reader.ReadBoolean();
             var output = ReadLengthPrefixedString(reader);
             var errorOutput = ReadLengthPrefixedString(reader);
+            if (!string.IsNullOrEmpty(errorOutput))
+            {
+                throw new InvalidOperationException();
```

Comment:
@AlekseyTs i went back and forth on this one.  This value will always be null / empty when run by our code.  But since this is also a network protocol other implementations in theory could provide a value.  

After thinking some more I'm going to move back to just ignoring this value.